pkg/client/client_test.go

```diff
@@ -431,7 +429,7 @@ var _ = Describe("Client", func() {
 				Expect(err).To(HaveOccurred())
 				// TODO(seans): Add test to validate the returned error. Problems currently with
 				// different returned error locally versus travis.
-			}, serverSideTimeoutSeconds)
```

Comment:
That does not seem the right way to move forward with and ensure the same behaviour.
See: https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-async-testing

Reply:
@camilamacedo86 can you check?

Reply:
@camilamacedo86 ginkgo honours timeouts like the one removed here if and only if the function passed to `It` accepts a `done` channel, otherwise it ignores it. 

Before this change the function passed to the `It` the removed timeout is for didn't have a `done` channel argument - so the timeout was completely ignored.

So removing the timeout here doesn't alter behavior.